src/components/transactions/TransactionDetails.vue

```diff
@@ -0,0 +1,202 @@
+<template>
```

Comment:
I'm not that happy with this component. Its purpose is to render the correct view for each type of transaction. 

1. There is a lot of repetition. I tried and failed to import the components dynamically based on the filename.
2. Each `XYZDetails` repeats the same classes (`tx__content__caption`, `tx__content__information` etc.). I feel this could be replaced with slot, or slot-scopes, but couldn't find a nice way to do it.